common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><i class='icon fa fa-times'></i></button>" +
+        "  <button class='close-modal dismiss' aria-hidden='true'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
Yes, all icon fonts should have `aria-hidden="true"`. In fact, this button should have a `title="Close"` as well so it's purpose is clear.